### DISCUSSION TITLE
Where should env_file be located?

### DISCUSSION THREAD

**Discussion by ajstaal** [hosted-domains/stack-deploy-action#3] [Q&A]
Hello and thanks for building this action! I was wondering if the `env_file` input variable is supposed to be a file that exists in the repo, because I was kinda hoping it could be a .env file living on my vps instead, which I don't want to commit to my repo as you can imagine.

**Comment by smashedr** [comment]
The `env_file` does point to a file located somewhere in the build working directory.

Currently my work around for your problem, is to create a separate repository containing my environment files and secrets that is private and have that repository checked out during the deploy and point to the env_file located in there.

This is an example workflow that does exactly that: https://github.com/cssnr/django5-boiler/blob/master/.github/workflows/deploy.yaml

Let me know if you have any more questions...
